RecoLocalTracker/SiPixelClusterizer/plugins/GPUClusterThresholds.h

```diff
@@ -0,0 +1,9 @@
+#ifndef RecoLocalTracker_SiPixelClusterizer_plugins_GPUClusterThresholds_h
+#define RecoLocalTracker_SiPixelClusterizer_plugins_GPUClusterThresholds_h
+
+struct GPUClusterThresholds {
+  const int32_t layer1;       // Cluster threshold in electrons for Layer 1
+  const int32_t otherLayers;  // Cluster threshold in electrons
+};
```

Comment:
Yes, but the whole point is that these are not constants, but rather they should be modifiable from config files.
Also, is this naming documented on http://cms-sw.github.io because I couldn't find it ?

Reply:
>  Also, is this naming documented on http://cms-sw.github.io because I couldn't find it ?

no, these are in the broad "common sense" category.

`k` is used as a prefix for (class/enum/global level constants) in different style guides, e.g. in google style https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
I know what it means, and also that it's in the Google C++ Style Guide. I also know of groups that follow the Google C++ Style Guide, but they documented it (I did not find the link, but they did the at DUNE). I mainly just meant, that if this is something I should use, I think it would useful to document it on the appropriate website.

Reply:
we do not have anything close to a coherent style of constant names, perhaps it's worth to propose something (outside the scope of this PR discussion). 
In the context of this PR I chose to type something explicit instead of saying "please define a named constant".